LIMS_IMAGE/web/templates/orders/order_history.html

```diff
@@ -1,24 +1,31 @@
 {% extends 'base.html' %}
+{% load static %}
 <!-- Not having this here makes the interpreter mad -->
-{% block title %}Client Home{% endblock %}
+{% block title %}Order History{% endblock %}
 <!-- Not having this here makes the interpreter mad -->
+{% block additionalStyles %}
+<link rel="stylesheet" href="{% static 'orders/css/order_history.css' %}" />
+<script src="{% static 'laboratory/js/sample_list_search.js' %}" defer></script>
+{% endblock %}
 <body>
-    <h1>Order Hisotry</h1>
     {% block content %}
-
-    {% if orders %}
-        <table>
-            <tr>
-                <th>Order Number</th>
-                <th>Submission Date</th>
+    <h1>Order Hisotry</h1>
+    <div class="container">
+        {% if orders %}
+            <table>
+                <tr>
+                    <th>Submission Date</th>
+                    <th>Order Number</th>
+                    <th>Arrival Date</th>
             </tr>
                 {% for order in orders %}
                 <tr>
+                    <td>{{order.submission_date}}</td>
                     <td><a href="{% url 'orders:order_page' order.order_number %}">{{order.order_number}}</a></td>
-                    <td>{{order.submission_date}}</td>                
```

Comment:
I think clients would care more about when their order was sent (submission date) rather than the day it arrives (arrival date)

Reply:
Should I remove the arrival date?
It is sorted by submission date.

Reply:
You should display submission date instead of or as well as the arrival date

Reply:
So currently I have the submission date, order number, and arrival date. 
![image](https://user-images.githubusercontent.com/62345938/160729624-85508bae-5510-4bd5-a0b3-0d441c0bbc15.png)

Reply:
Oh my bad! I totally missed that